lib/screens/profile/contact_support.dart

```diff
@@ -7,152 +11,468 @@ class ContactSupport extends StatefulWidget {
+      },
+    );
+  }
+}
+
+class DeviceSpecListLoop extends StatelessWidget {
+  const DeviceSpecListLoop({
+    Key? key,
+    required this.data,
+  }) : super(key: key);
+
+  final List<ListItem> data;
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: [
+        for (var item in data)
+          Column(
+            children: <Widget>[
+              Padding(
+                padding: const EdgeInsets.symmetric(vertical: 11.0),
+                child: Row(
+                  mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                  crossAxisAlignment: CrossAxisAlignment
+                      .start, //control the LH bar vertical alignment
+                  children: <Widget>[
+                    Flexible(
+                      fit: FlexFit.tight,
+                      child: Text(
+                        item.title,
+                        style: const TextStyle(
+                          fontSize: 17,
+                          fontWeight: FontWeight.w500,
+                          color: Color.fromRGBO(112, 124, 142, 1),
+                        ),
+                      ),
+                    ),
+                    Flexible(
+                      fit: FlexFit.tight,
+                      child: Text(
+                        item.content ?? 'N/A',
+                        textAlign: TextAlign.right,
+                        style: const TextStyle(
+                          fontSize: 17,
+                          fontWeight: FontWeight.w500,
+                          color: Color.fromRGBO(254, 254, 254, 1),
+                        ),
+                      ),
+                    ),
+                  ],
+                ),
+              ),
+              const Divider(
+                height: 1,
+                color: Color.fromRGBO(204, 221, 255, 0.1),
+              ),
+            ],
           ),
-        ),
-        child: FittedBox(
-          child: Text('Send'),
-        ),
+      ],
+    );
+  }
+}
+
+class SendingResultPage extends StatelessWidget {
+  final GlobalKey _widgetKey = GlobalKey();
+  final String sendErrorMsgTitle = 'Couldn’t send you message';
+  final String sendErrorMsgContent =
+      'Something went wrong. Check your internet connection and try again.';
+  final String sendSuccessMsgTitle = 'Thank you for your feedback!';
+  final String sendSuccessMsgContent =
+      'We appreciate all feedback and we will address the issue as soon as possible.';
+  final String successBtnMsg = 'Done';
+  final String errorBtnMsg = 'Send again';
+  final bool sendingResult;
+
+  SendingResultPage({
+    super.key,
+    this.sendingResult = true,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return Expanded(
+      child: Column(
+        children: <Widget>[
+          Expanded(
+            child: Padding(
+              padding: EdgeInsets.only(
+                  top: MediaQuery.of(context).size.height * 0.3),
```

Comment:
Why is the padding based on the window height?

Reply:
I was trying to achieve to make the msg widget to be center of the screen, dispite the appbar and the bottom and the notification bar, so I teyied to implement the width with the widget's height 
           for example: top: (MediaQuery.of(context).size.height - Widget's height) / 2

but in order to do that, I don't have sufficient skill and knowledge, plus the approaches to do that are not so easy or will used up a lot of resource (usually applies to animation which aims to re-size the layout based on the changes)  

base the things mentioned above, the end-product turns out like this :\

refs: https://stackoverflow.com/questions/49307677/how-to-get-height-of-a-widget